src/Application/Model/LogEntry.php

```diff
@@ -126,6 +126,12 @@ class LogEntry extends Model {
 				'multiple' => '{user_name} abortet checking {tickets}.',
```

Comment:
Unrelated, but "abortet" might also not be an English word ;)

Reply:
Fixed in 39020caf.